lua/mapvote/server/modules/map_vote.lua

```diff
@@ -44,7 +46,7 @@ function MapVote.Start( length )
 
     local mapsInVote = {}
 
-    for _, map in RandomPairs( maps ) do
+    for _, map in Shuffle( maps ) do
```

Comment:
The main point of this was for NPC Servers so it would sort alphabetically and people could memorize the position of maps, maybe instead the map limit could be disregarded and set to the amount of included maps if the shuffle is off

Reply:
I think implementing a sort like I mentioned in my other comment would be much more straight forward from the perspective a user of this addon. It is a little bit convoluted to have a config option that causes another config option to change. We would also have to take into account the case where there are more maps than can fit on the screen.

Reply:
I can also implement this if you would like.

Reply:
I'm not sure exactly what you mean or how to go about coding it, but if you could that would be great, thank you!